### ISSUE TITLE
confusing documentation about cross-compilation

### ISSUE THREAD

**Issue by umlaeute** [pure-data/pd-lib-builder#57]
The tips-and-tricks starts with a chapter "cross-compiling on linux x86_64 for other platforms":

https://github.com/pure-data/pd-lib-builder/blob/df9eb41d68598180bdb532a8fbe5b7bd310796c0/tips-tricks.md#L5-L7

why does it say "linux **x86_64**"? the cross-build toolchains are independent of the host. e.g. (in Debian) there are "i686-linux-gnu" packages available for the `arm64` architecture, and afaics you can install mingw on your raspberry.

i suggest to just drop the architecture specifier and call it "*cross-compiling on linux for other platforms and architectures*"

**Comment by katjav** [comment]
In commit 2d3b7f4 I changed the title from

> cross-compiling W32 binaries from linux

to

> cross-compiling on linux x86_64 for other platforms

to reflect what the text currently covers. And the text covers cross tools that I recently verified to work on a Debian Buster - based x86_64 system (MX19). Raspbian doesn't seems to have such a complete set, is what I remember from last time checking it. But you're right, the title can be general. And the text should be more explicit about what is verified. I'll change that.
